distage/distage-roles/src/main/scala/com/github/pshirshov/izumi/distage/roles/launcher/RoleStarterImpl.scala

```diff
@@ -21,16 +21,22 @@ class RoleStarterImpl
   executors: Set[ExecutorService],
   integrations: Set[IntegrationComponent],
   logger: IzLogger,
+  resourceCollection: ResourceCollection = new NullResourceCollection
```

Comment:
Creating your own dependencies is the problem DI is supposed to solve in the first place, please don't do that...